pennylane_lightning/core/_serialize.py

```diff
@@ -133,6 +161,11 @@ def __init__(
 
             self._mpi_manager = lightning_ops.MPIManager
 
+        else:
```

Comment:
when will this be encountered?

Reply:
Good question. In the case of adding a new device. However, the name is defined from the device construction. :thinking: 
I will remove the `else` condition.